packages/cli/src/user/user.service.ts

```diff
@@ -26,15 +28,14 @@ export class UserService {
 	}
 
 	static async generatePasswordResetUrl(user: User): Promise<string> {
-		user.resetPasswordToken = uuid();
-		const { id, resetPasswordToken } = user;
-		const resetPasswordTokenExpiration = Math.floor(Date.now() / 1000) + 7200;
-		await Db.collections.User.update(id, { resetPasswordToken, resetPasswordTokenExpiration });
-
+		const { id } = user;
+		const token = jwt.sign({ sub: id }, config.getEnv('userManagement.jwtSecret'), {
+			expiresIn: '1h',
```

Comment:
I think 1 hour is a very short time windows to reset a password, can we maybe extend it to 1d? Should still be secure enough.